mlflow/tensorflow.py

```diff
@@ -1,4 +1,17 @@
-"""MLflow integration for Tensorflow."""
+"""MLflow integration for Tensorflow.
+
+Manages logging and loading Tensorflow models as Python Functions. You are expected to save your own
+``saved_models`` and pass their paths to ``log_saved_model()`` 
+so that MLflow can track the models. 
+
+In order to load the model to predict on it again, you can call
+``estimator = mlflow.pyfunc.load_pyfunc(saved_model_dir)``, followed by 
```

Comment:
The return value of `mlflow.pyfunc.load_pyfunc()` is a pyfunc model, so let's use `model` instead of `estimator` here